index.html

```diff
@@ -0,0 +1,33 @@
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>simpleweb</title>
+    </head>
+    <body>
+        <header>
+        <div class="toolbar">
+            <div class="logo">My Logo</div>
+            <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
+        </div>
+        </header>
+    <section>
+         <div> 
+        <h1 class="header">I'm Mariane<br>Glad to see you!</h1></div>    
+      <div>  <p class="about">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    </section>
+        <footer class="contact">
+            <p class="connect">LET'S CONNECT</p>
+            <ul class="list">
+                <li class="list-item"><img src="images/tweeter.png" alt="follow me on twitter" width="20" height="20" ></li>
+                <li class="list-item"><img src="images/linkedin.png" alt="follow me on linkedin" width="20" height="20" ></li>
+                <li class="list-item"><img src="images/medium.png" alt="follow me on medium" width="20" height="20" ></li>
+                <li class="list-item"><img src="images/github.png" alt="follow me on github" width="20" height="20" ></li>
+                <li class="list-item"><img src="images/angelist.png" alt="follow me on angellist" width="20" height="20" ></li>
+            </ul>
```

Comment:
Good job for using the list in here 👌 and you can improve it by putting the `img` tags in the `a` tag, because every icon is a link to a social media profile. So please make sure to fix this.

Example
 ```Javascript
<li class="list-item">
  <a href="https://twitter.com/your-twitter-handler">
    <img src="images/tweeter.png" alt="follow me on twitter" width="20" height="20" />
  </a>
</li>
```